lib/newgistics/customer.rb

```diff
@@ -1,6 +1,6 @@
 module Newgistics
   class Customer
-    include Virtus.model
+    include Newgistics::Model
```

Comment:
I wasn't sure if we should go with including a module or inheriting from a base class, any thoughts?

Reply:
I think I personally like inheritance. I think it would make sense for all of our models to inherit from the `Newgistics::Model`. But do you have any reservations?

Reply:
The only thing I can think of, is we we'd need to make the base class abstract, because it wouldn't make sense to instantiate it and abstract class support in ruby is weird, other than that I'm good with either approach